pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -615,6 +615,9 @@ func (o *GinkgoRunSuiteOptions) filterOutRebaseTests(restConfig *rest.Config, te
 		// compare https://github.com/kubernetes/kubernetes/pull/123405
 		// which changed the healthz handler name
 		`[sig-api-machinery] health handlers should contain necessary checks`,
+		`PersistentVolumes GCEPD.*`,
```

Comment:
Down below this looks like a strings.Contains, I don't think that will work with the .* wildcard, just need to remove that.